src/CMakeLists.txt

```
@@ -20,10 +20,10 @@ set(HOUDINI_RELATIVE_PALLADIO_PATH "${HOUDINI_RELATIVE_PACKAGES_PATH}/palladio")
 
 ### versioning
 
-set(PLD_VERSION_MAJOR 1)
-set(PLD_VERSION_MINOR 10)
+set(PLD_VERSION_MAJOR 2)
+set(PLD_VERSION_MINOR 0)
 set(PLD_VERSION_PATCH 0)
-set(PLD_VERSION_PRE "-dev.0") # set to empty string for final releases
+set(PLD_VERSION_PRE "-Beta-1") # set to empty string for final releases
```

let's keep with the semver syntax for beta:

```suggestion
set(PLD_VERSION_PRE "-beta.1") # set to empty string for final releases
```

see how we did it previously, e.g. https://github.com/Esri/palladio/blob/d6feb8c18706a58220c5501b3d7f794d57c85a16/src/CMakeLists.txt#L48